airgun/entities/subscription.py

```diff
@@ -71,7 +64,12 @@ def read_delete_manifest_message(self):
         """Read message displayed on 'Confirm delete manifest' dialog"""
         view = self.navigate_to(self, 'Delete Manifest Confirmation')
         view.wait_animation_end()
-        return view.message.read()
+        delete_message = view.message.read()
+        view.cancel_button.click()
+        manage_view = ManageManifestView(self.browser)
+        if manage_view.is_displayed:
+            manage_view.close_button.click()
```

Comment:
imo some comment explaining why do we need that would be helpful (if i got you right - you're closing modal window to unblock further auto steps, aren't you?)

Reply:
If we do not close that dialog, The further navigation will be broken, especially when we want to navigate to subscription page again  as it will find that we are at that page already and fail.

Reply:
Sure. I meant comment in the code :)